examples/COCO/scripts/run2auto.batch

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
and make sure you only have one slurm script `run_auto.batch`...

Reply:
No problem

Reply:
I will change the slurm script a little bit and provide to support various number of GPU more easily.